### ISSUE TITLE
Why the loss function formula is different in PPO code and paper?

### ISSUE THREAD

**Issue by bas1003** [openai/baselines#1115]
Hi,

In code, calculate the total loss like this,
`loss = pg_loss - entropy * ent_coef + vf_loss * vf_coef`
While in paper PPO, 
![image](https://user-images.githubusercontent.com/67232848/85228202-447ef680-b40c-11ea-8fa5-e5e84cda68db.png)
(Eqn 10, https://arxiv.org/abs/1707.06347)

Why the loss function formula is different in PPO code and paper? Can someone explain why like that?
Thank you.

**Comment by BingyuZhou** [comment]
Because in the paper, they are maximize the loss, while in code we usually minimize loss.
